Slim/Control/Queries.pm

```diff
@@ -895,6 +897,14 @@ sub albumsQuery {
 				}
 			}
 
+			if ( $tags =~ /2/ ) {
+				$groupsSth ||= $dbh->prepare_cached("SELECT COUNT(DISTINCT COALESCE(work,1) || COALESCE(grouping,1) || COALESCE(performance,1)) FROM tracks WHERE tracks.album=?");
```

Comment:
`||` does concatenate strings. How would that count results? This could potentially be `1111` for various actual counts (11 + 1 + 1, or 1+1+11)?

Reply:
Oh, it's counting those strings, not summarizing them.